packages/fluentui/react-northstar/src/themes/teams-dark/components/Dropdown/dropdownVariables.ts

```diff
@@ -0,0 +1,6 @@
+import { DropdownVariables } from '../../../teams/components/Dropdown/dropdownVariables';
+
+export default (siteVars: any): Partial<DropdownVariables> => ({
+  loadingMessageBackgroundColor: siteVars.colors.dark,
```

Comment:
Since loading and noResults messages look similar to listItem, why not to use the same default variable there as in default background color for listItem -`listItemBackgroundColor: 'transparent',` (see packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownVariables.ts)
?

Reply:
Changed

Reply:
Weren't these variables removed for a reason? I am not sure how it works right now, but maybe @mnajdova knows :D

Reply:
@silviuaavram Don't think so, in the dark theme and HC the loading slot is white for now

Reply:
Changed the default variables